.travis.yml

```diff
@@ -4,10 +4,10 @@ python:
 services:
 - docker
 before_install:
-- docker pull univa/tortuga-build-kit:v7.0.1-latest
+- docker pull univa/tortuga-build-kit:7.0.2-devel-latest
```

Comment:
I think this needs to be changed to `7.0.2` as the `-devel` branch is going away soon.